mdakits/sasa/metadata.yaml

```
@@ -0,0 +1,53 @@
+project_name: mdakit_sasa
+
+## List(str): a link to the authors file (preferred) or a list of authors 
+authors:
+  - https://github.com/pegerto/mdakit_sasa/blob/main/AUTHORS.md
+
+maintainers:
+  - pegerto
+
+description:
+    This MDAKit allows enables the calculation of Solvant Accessible Surface Area ( SASA) with MDAnalysis.
+
+## List(str): a list of keywords which describe the mdakit
+keywords:
+  - Sasa
+  - FreeSasa
```

```suggestion
  - SASA
  - FreeSASA
```
Small gramatical thing